src/library.coffee

```diff
@@ -80,8 +82,10 @@ componentsFromDirectory = (directory, config, callback) ->
 
 normalizeConfig = (config) ->
   config = {} if not config
+  namespace = config.name or null
```

Comment:
This looks inconsistent, using `.name` where everywhere else it is `.namespace`? is this because respecting the standard package.json property? If so, needs a comment saying this.